srcs/requirements/backend/Dockerfile

```
@@ -1,4 +1,18 @@
-FROM alpine
+FROM node:20-slim
 
-# Set a default command to keep the container alive
-CMD ["sleep", "infinity"]
+# Required by latest Prisma version
+RUN apt-get update -y && apt-get install -y openssl
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)